src/patternfly/components/Alert/examples/alert-warning-example.hbs

```diff
@@ -1,33 +1,44 @@
- {{#> alert pf-c-alert--Modifier="pf-m-warning" pf-c-alert--AriaLabel="Warning Notification"}}
-<div aria-hidden="true" class="pf-c-alert__icon">
-  <i class="fas fa-exclamation-triangle"></i>
-</div>
-<div class="pf-c-alert__body">
-  <h4 class="pf-c-alert__title">
-    <span class="pf-u-sr-only">Warning: </span> Warning notification title</h4>
+{{#> alert pf-c-alert--modifier="pf-m-warning" pf-c-alert--attribute='aria-label="Warning Notification"'}}
+  {{#> alert-icon pf-c-alert-icon--warning="true"}}
+  {{/alert-icon}}
+  {{#> alert-body}}
+    {{#> alert-title}}
+      {{#> sr sr--modifier="pf-u-sr-only"}}
+        Warning:
+      {{/sr}}
+      Warning notification title
+    {{/alert-title}}
     <p>This is a description of the notification content.</p>
-</div>
+  {{/alert-body}}
 {{/alert}}
 <br>
-<br> {{#> alert pf-c-alert--Modifier="pf-m-warning" pf-c-alert--AriaLabel="Warning Notification"}}
-<div aria-hidden="true" class="pf-c-alert__icon">
-  <i class="fas fa-exclamation-triangle"></i>
-</div>
-<div class="pf-c-alert__body">
-  <h4 class="pf-c-alert__title">
-    <span class="pf-u-sr-only">Warning: </span> Warning notification title</h4>
-</div>
-<div class="pf-c-alert__action">
-  {{#> button btnClass="pf-m-secondary"}} Button {{/button}}
-</div>
+{{#> alert pf-c-alert--modifier="pf-m-warning" pf-c-alert--attribute='aria-label="Warning Notification"'}}
+  {{#> alert-icon pf-c-alert-icon--warning="true"}}
+  {{/alert-icon}}
+  {{#> alert-body}}
+    {{#> alert-title}}
+      {{#> sr sr--modifier="pf-u-sr-only"}}
+      Warning:
+      {{/sr}}
+      Warning notification title
+    {{/alert-title}}
+  {{/alert-body}}
+  {{#> alert-action pf-c-alert--attribute="pf-c-alert__action"}}
+    {{#> button btnClass="pf-m-secondary"}}
+      Button 
+    {{/button}}
+  {{/alert-action}}
 {{/alert}}
 <br>
-<br> {{#> alert pf-c-alert--Modifier="pf-m-warning" pf-c-alert--AriaLabel="Warning Notification"}}
-<div aria-hidden="true" class="pf-c-alert__icon">
-  <i class="fas fa-exclamation-triangle"></i>
-</div>
-<div class="pf-c-alert__body">
-  <h4 class="pf-c-alert__title">
-    <span class="pf-u-sr-only">Warning: </span> Warning notification title</h4>
-</div>
+{{#> alert pf-c-alert--modifier="pf-m-warning" pf-c-alert--attribute='aria-label="Warning Notification"'}}
+  {{#> alert-icon pf-c-alert-icon--warning="true"}}
+  {{/alert-icon}}
+  {{#> alert-body}}
+    {{#> alert-title}}
+      {{#> sr sr--modifier="pf-u-sr-only"}}
```

Comment:
once you update the sr.hbs you can remove the modifier here.

Reply:
done